tide_core.install

```diff
@@ -378,3 +379,72 @@ function tide_core_update_10012() {
   $shield_config = \Drupal::configFactory()->getEditable('shield.settings');
   $shield_config->set('paths', $paths_string)->save();
 }
+
+/**
+ * Extend custom filters to all content types.
+ */
+function tide_core_update_10013() {
+  $content_types = ['landing_page', 'publication', 'publication_page', 'event', 'grant', 'tide_search_listing', 'news'];
```

Comment:
These node types belong to different modules, so you shouldn’t put them all in tide_core. Instead, add each one to its respective module.

Reply:
Hey @vincent-gao , I've updated my pr as you requested.